src/app/(site)/tickets/page.tsx

```diff
@@ -1,294 +1,85 @@
-          <p>Stora salongen idag, Onsdag 17.00</p>
-        </div>
-      </div>
+  useEffect(() => {
+    const fetchData = async () => {
+      const res = await fetch('http://localhost:3000/api/tickets/64536ba84157b856e74f0a37', {
+        next: { revalidate: 1 },
+      });
+      const payload = await res.json();
 
-      <h2 className="header-two">Välj antal biljetter</h2>
-      <div className="number-of-tickets-container">
-        <div
-          onClick={() =>
-            setTicketCount(ticketCount != 0 ? ticketCount - 1 : ticketCount)
-          }
-          className="plus-minus"
-        >
-          -
-        </div>
-        <div className="ticket-count">{ticketCount}</div>
-        <div
-          onClick={() => setTicketCount(ticketCount + 1)}
-          className="plus-minus"
-        >
-          +
-        </div>
-      </div>
+      let numberOfSeats = 48;
+      let initialSeatsArray = [];
+      let bookedSeatsArray: number[] = payload.map((ticket: { status: number; seat: number }) => {
+        return ticket.seat;
+      });
 
-      <h2 className="header-two">Välj platser</h2>
+      for (let i = 1; i <= numberOfSeats; i++) {
+        if (bookedSeatsArray.includes(i)) {
+          initialSeatsArray.push({ status: Seat.booked, seatNumber: i });
+        } else {
+          initialSeatsArray.push({ status: Seat.available, seatNumber: i });
+        }
+      }
+      setSeatsArray(initialSeatsArray);
+    };
+    fetchData();
+  }, []);
 
-      <div className="movie-screen-seat-container">
-        <div className="movie-screen-container">
-          <div className="movie-screen-left"></div>
-          <div className="movie-screen-middle"></div>
-          <div className="movie-screen-right"></div>
-        </div>
-
-        <div className="seat-container">
-          {MOCKSCREENING.map((seat) => {
-            return (
-              <div
-                key={seat.place}
-                className={
-                  seat.available ? 'seat-available' : 'seat-not-available'
-                }
-              >
-                {seat.place}
-              </div>
-            );
-          })}
-        </div>
-      </div>
+  const [ticketCount, setTicketCount] = useState(0);
+  const [seatsArray, setSeatsArray] = useState<[] | { status: Seat; seatNumber: number }[]>([]);
 
-      <div className="choose-seat-container">
-        <div className="choosed-seat">☑️</div>
-        <p>Vald plats</p>
-      </div>
-      <div className="choose-seat-container">
-        <div className="booked-seat"></div>
-        <p>Upptagen</p>
-      </div>
+  const [continueGuest, setContinueGuest] = useState(false);
 
-      <div className="divider"></div>
+  const handleGuest = () => {
+    setContinueGuest(true);
+  };
 
-      <div className="total-sum-container">
-        <h2>Att betala:</h2>
-        <h2>{ticketCount * 150}kr</h2>
-      </div>
+  if (continueGuest) {
+    return <GuestTickets tickets={ticketCount} />;
```

Comment:
What do you need ticketCount for here? I think we need to create state for the booking with screeningId, seatNumber and email instead. That is what we need to create a ticket.

Reply:
I think we need to pass ticketCount to payment, so customer can se total price at the end?

Reply:
With React we can save it in state during the ticket booking.